lib/manager/npm/update/dependency/index.ts

```diff
@@ -3,6 +3,55 @@ import { logger } from '../../../../logger';
 import { matchAt, replaceAt } from '../../../../util/string';
 import type { UpdateDependencyConfig } from '../../../types';
 
+function replaceAsString(
+  parsedContents: any,
```

Comment:
I think we already have a type for this, so we won't need `any` here.

Reply:
`PackageJson`?